cypress/integration/bonus_spec.js

```diff
@@ -0,0 +1,21 @@
+describe("Bonus: Vowel Remover", () => {
+  it("removes vowels excluding y when the checkbox is not checked", () => {
```

Comment:
I noticed in https://github.com/joinpursuit/adding_elements_to_the_dom_lab that there's a `cy.visit` command in a test. Should I be adding that to these?

cc @mikeboyle

Reply:
Yeah, I think the past labs have hidden that `cy.visit` in the `commands.js` file. Example: https://github.com/joinpursuit/Pursuit-Core-CSS-Intro-Lab/blob/master/cypress/support/commands.js

Reply:
The `cy.visit` command has been tucked into the commands.js file: https://github.com/joinpursuit/combining-html-and-javascript-plus-dom/blob/master/cypress/support/commands.js